server/data/mutations/todo.js

```diff
@@ -0,0 +1,113 @@
+/* @flow */
+
+import {
+  GraphQLNonNull,
+  GraphQLString,
+  GraphQLID,
+} from 'graphql';
+
+import {
+  mutationWithClientMutationId,
+  offsetToCursor,
+  fromGlobalId,
+} from 'graphql-relay';
+
+import { userType } from '../types/userType';
+import { TodoEdge, todoType } from '../types/todoType';
+import {
+  addTodo,
+  getTodo,
+  getNumTodos,
+  getViewer,
+  removeTodo,
+  updateTodo,
+} from '../database';
+
+const addTodoMutation = mutationWithClientMutationId({
+  name: 'AddTodo',
+  inputFields: {
+    content: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+  },
+  outputFields: {
+    newTodoEdge: {
+      type: TodoEdge,
+      resolve: async ({ localTodoId }) => {
+        const todo = await getTodo(localTodoId);
+        const numTodos = await getNumTodos();
+        return {
+          cursor: offsetToCursor(numTodos - 1),
```

Comment:
can we add a comment noting that using offsetToCursor with numTodos is an alternative workaround for cursorForObjectInConnection (https://github.com/graphql/graphql-relay-js/issues/29)? because i think relay tutorials generally use cursorForObjectInConnection.